src/renderer/widgets/SendAssetModal/ui/components/ActionSteps/InitOperation.tsx

```diff
@@ -107,8 +153,15 @@ export const InitOperation = ({
         nativeToken={nativeToken}
         addressPrefix={addressPrefix}
         fee={fee}
+        xcmFee={xcmFee}
```

Comment:
Maybe create object prop with keys like xcmFee, xcmWeight so it would be more compact and you can pass it like xcmPros={xcmFee, xcmWeight}